OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseInfoActivity.java

```diff
@@ -24,12 +23,11 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onStart() {
         super.onStart();
 
-        String pathId = getIntent().getStringExtra(EXTRA_PATH_ID);
-        String url = environment.getConfig().getCourseDiscoveryConfig()
+        final String pathId = getIntent().getStringExtra(EXTRA_PATH_ID);
+        final String url = environment.getConfig().getCourseDiscoveryConfig()
                 .getCourseInfoUrlTemplate()
                 .replace("{" + EXTRA_PATH_ID + "}", pathId);
-        WebView webview = (WebView) findViewById(R.id.webview);
-        webview.loadUrl(url);
+        loadUrlBasedOnOsVersion(url);
```

Comment:
**nit**
**loadUrlBasedOnOsVersion** doesn't look good here. simple **loadUrl** will be good.